crates/precompile/src/bls12_381.rs

```diff
@@ -1,18 +1,27 @@
 use crate::PrecompileWithAddress;
 
 mod g1;
+#[cfg(feature = "blst")]
 pub mod g1_add;
+#[cfg(feature = "blst")]
 pub mod g1_msm;
 mod g2;
+#[cfg(feature = "blst")]
 pub mod g2_add;
+#[cfg(feature = "blst")]
 pub mod g2_msm;
+#[cfg(feature = "blst")]
 pub mod map_fp2_to_g2;
+#[cfg(feature = "blst")]
 pub mod map_fp_to_g1;
-pub mod msm;
+#[cfg(feature = "blst")]
 pub mod pairing;
 mod utils;
+pub mod bls12_381_const;
```

Comment:
Final thing, move bls12_381_const to `src/` folder and remove all `#[cfg(feature = "blst")]` from this file. This would work as the main cfg feature from lib.rs is still active.

Reply:
Just made these changes and pushed! do check it when free!